Tests/ListFileInfoTests.m

```diff
@@ -61,7 +61,7 @@ - (void)testListFileInfo {
         XCTAssertEqual(fileInfo.uncompressedSize, expectedFileSize, @"Incorrect uncompressed file size");
         
         // Test Compression method
-        UZKCompressionMethod expectedCompressionMethod = ((NSNumber *)expectedCompressionMethods[fileInfo.filename]).integerValue;
+        UZKCompressionMethod expectedCompressionMethod = ((NSNumber *)expectedCompressionMethods[fileInfo.filename]).intValue;
```

Comment:
FYI, this was fixing a compiler warning in Xcode 11. I didn't mean to include it, and have removed it from my branch, if you could please pull that change in.